ball_detect/main.c

```diff
@@ -1,8 +1,10 @@
 #include <avr/io.h>
```

Comment:
Please separate these includes into following groups:

 1. standard C library
 2. thirdparty libraries
 3. local includes (if any)

Each group should also be sorted alphabetically.